### PR TITLE
Add infoenergia reports

### PR DESCRIPTION
**Pull request by marta197**
- Add `api/InfoenergiaReport/{contract_id}` endpoint to extract infoenergia reports from mongo.

Request example:
```
 headers: {
    'Authorization': token
  }
```

### CODE REVIEW

File: heman/api/infoenergia/__init__.py [hunk 1]
```diff
@@ -0,0 +1,51 @@
+from datetime import datetime
+from dateutil.relativedelta import relativedelta
+import json
+import time
+
+from flask import current_app, jsonify, request, Response
+from pymongo import ASCENDING
+
```
Comment (juan-pe) [inline heman/api/infoenergia/__init__.py:5]: `datetime`, `relativedelta`, `time`, `request` and `ASCENDING` imports are not used :fearful: :laughing: :laughing:

**Review by juan-pe — CHANGES_REQUESTED** [review]
Easy right?? :smile: 
if you can delete the imports that aren't used, this code is perfect!

thanks a lot!

File: heman/api/infoenergia/__init__.py (hunk 1 above)
Comment (marta197) [reply heman/api/infoenergia/__init__.py:5]: Good you noticed it!! It'll be fixed on the next commit. Thank you! :smile:

**Review by juan-pe — APPROVED** [review]
:+1: fantastic!

**Pull request merged**